infra/gcb/build_project.py

```diff
@@ -357,7 +357,12 @@ def dataflow_post_build_steps(project_name, env):
 
   steps.append({
       'name': 'gcr.io/oss-fuzz-base/base-runner',
-      'env': env,
+      'env': env + [
+          'COLLECT_DFT_TIMEOUT=2h',
+          'DFT_FILE_SIZE_LIMIT=65535',
```

Comment:
these are overwriting the defaults in https://github.com/google/oss-fuzz/blob/8faac99098ad28db35d12376e602294772c006c7/infra/base-images/base-runner/dataflow_tracer.py#L27